.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a bug report to help us improve eksctl
+
+---
+
+**What happened?**
+A description of actual behvaior (with error messages).
+
+**What you expected to happen?**
+A clear and concise description of what the bug is.
+
+**How to repoduce it?**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Anything else we need to know?**
+Hardware, OS and screenshots (if applicable)
```

Comment:
I would take out 'Hardware' and 'screenshots'. Perhaps we could ask about the OS, whether they installed binary or compiled it, and how are their AWS credentials configured (namely MFA and stuff). Wdyt?

Reply:
Yeah i think thats a good idea, i have changed.